Description: Use /usr/bin/wish instead of /usr/local/bin/wish.
Author: Benjamin Drung <bdrung@debian.org>

--- a/src/dosfop/tcl/browse_tcl
+++ b/src/dosfop/tcl/browse_tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/browse_tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 option readfile $env(DOSFOP)/defaults/options.data
--- a/src/dosfop/tcl/dosfop_tcl
+++ b/src/dosfop/tcl/dosfop_tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/dosfop_tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 # At least the DOSFOP environment variable has to be present for execution!
@@ -13,8 +13,8 @@
 
 if {[info exists env(WISH)] == 0} then {
     puts "WISH environment variable does not exist"
-    puts "using '/usr/local/bin/wish' as default"
-    set env(WISH) "/usr/local/bin/wish"
+    puts "using '/usr/bin/wish' as default"
+    set env(WISH) "/usr/bin/wish"
 }
 
 option readfile $env(DOSFOP)/defaults/options.data
--- a/src/dosfop/tcl/filebrowser_tcl
+++ b/src/dosfop/tcl/filebrowser_tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/filebrowser_tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 # Parameter : Name of calling tcl-application 
 
--- a/src/dosfop/tcl/globalConfig_tcl
+++ b/src/dosfop/tcl/globalConfig_tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/globalConfig_tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 option readfile $env(DOSFOP)/defaults/options.data
--- a/src/dosfop/tcl/pureDosfop.tcl
+++ b/src/dosfop/tcl/pureDosfop.tcl
@@ -1,4 +1,4 @@
-#! /usr/local/bin/wish -f
+#! /usr/bin/wish -f
 ## managing dosfop tools without window system
 
 source $env(DOSFOP)/tcl/pureExec.tcl
--- a/src/dosfop/tcl/startConsistencyCheck.tcl
+++ b/src/dosfop/tcl/startConsistencyCheck.tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/startConsistencyCheck.tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 wm withdraw .
--- a/src/dosfop/tcl/startDosfop.tcl
+++ b/src/dosfop/tcl/startDosfop.tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/startDosfop.tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 wm withdraw .
--- a/src/dosfop/tcl/startMakeinfo.tcl
+++ b/src/dosfop/tcl/startMakeinfo.tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/startMakeinfo.tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 wm withdraw .
--- a/src/dosfop/tcl/startTexi2dvi.tcl
+++ b/src/dosfop/tcl/startTexi2dvi.tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/startTexi2dvi.tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 wm withdraw .
--- a/src/dosfop/tcl/startTexi2html.tcl
+++ b/src/dosfop/tcl/startTexi2html.tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/startTexi2html.tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 wm withdraw .
--- a/src/dosfop/tcl/structureConfig_tcl
+++ b/src/dosfop/tcl/structureConfig_tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/structureConfig_tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 # Parameter: Name of Structure
--- a/src/dosfop/tcl/structuresConfigGlobal_tcl
+++ b/src/dosfop/tcl/structuresConfigGlobal_tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/structuresConfigGlobal_tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 # Parameter : Location
--- a/src/dosfop/tcl/subsystemConfig_tcl
+++ b/src/dosfop/tcl/subsystemConfig_tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish -f
+#!/usr/bin/wish -f
 # $Id: /opal/ocs/tags/2.4/src/dosfop/tcl/subsystemConfig_tcl 7677 2010-09-30T15:50:09.291274Z florenz  $
 
 # Parameter: Name of Subsystem
